certs.yml

```diff
@@ -0,0 +1,111 @@
+- hosts: mail.hashbang.sh
+  become: True
+  vars_files:
+    - files/certs/hashbang.sh.yml
+    - files/certs/hashbang.sh.key
+  tasks:
+    - copy:
+        content: "{{ item.value }}"
+        dest: "postfix/config/certs/{{ item.key }}"
+        mode: 0444
+
+      with_dict:
+        cacert.pem: "{{   ca }}"
+        server.crt: "{{ cert }}"
+        server.key: "{{  key }}"
+
+      loop_control:
+        label: "{{ item.key }}"
+
+      notify: restart postfix
+
+  handlers:
+    - name: restart postfix
+      service:
+        name:  docker-postfix
+        state: restarted
+
+- hosts: irc_servers
+  become: False
+  vars_files:
+    - files/certs/hashbang.sh.yml
+    - files/certs/hashbang.sh.key
+  tasks:
+    - copy:
+        content: "{{ item.value }}"
+        dest: "unrealircd/certs/{{ item.key }}"
+        mode: 0400
```

Comment:
Is there a reason this specific one is `0400`?

Reply:
I kept the permissions that were already there.

Reply:
Alright, guess there's no issue with that.